container/metadata.csv

```diff
@@ -0,0 +1,26 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+container.uptime,gauge,,second,,The container uptime,0,container,uptime
+container.cpu.usage,rate,,nanosecond,,The container total CPU Usage,0,container,cpu_total
+container.cpu.user,rate,,nanosecond,,The container userspace CPU usage,0,container,cpu_user
+container.cpu.system,rate,,nanosecond,,The container system CPU usage,0,container,cpu_system
+container.cpu.throttled.time,rate,,nanosecond,,The total cpu throttled time,0,container,cpu_throttled_time
+container.cpu.throttled.periods,rate,,,,The number of periods during which the container was throttled,0,container,cpu_throttled_periods
+container.cpu.shares,gauge,,,,The number of CPU shares assigned to the container,0,container,cpu_shares
+container.cpu.limit,gauge,,nanosecond,,The maximum CPU time available to the container,0,container,cpu_limit
+container.memory.usage,gauge,,byte,,The container total memory usage,0,container,mem_usage
+container.memory.kernel,gauge,,byte,,The container kernel memory usage,0,container,mem_kernel_usage
+container.memory.limit,gauge,,byte,,The container memory limit,0,container,mem_limit
+container.memory.soft_limit,gauge,,byte,,The container memory soft limit,0,container,mem_soft_limit
+container.memory.rss,gauge,,byte,,The container RSS usage,0,container,mem_rss
+container.memory.cache,gauge,,byte,,The container cache usage,0,container,mem_cache
+container.memory.swap,gauge,,byte,,The container swap usage,0,container,mem_swap
+container.memory.oomevents,gauge,,byte,,The number of OOM events triggered by the container,0,container,mem_oomevents
+container.memory.working_set,gauge,,byte,,The container working set usage,0,container,mem_workingset
+container.memory.commit,gauge,,byte,,The container commit memory usage,0,container,mem_commit
+container.memory.commit.peak,gauge,,byte,,The container pack commit memory usage,0,container,mem_commit_peak
+container.io.read,rate,,byte,,The cache amount used,0,container,io_read
```

Comment:
from this line to the bottom of the doc i think the descriptions don't match the metric name